jenkins-integration/scripts/foss_install.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+SYSTEST_CONFIG=$1
+SSH_KEYFILE=$2
+
+bundle exec beaker             \
+  --config "$SYSTEST_CONFIG"   \
+  --helper beaker/helper.rb    \
+  --tests beaker/install/foss/ \
+  --preserve-hosts             \
+  --no-color                   \
+  --xml                        \
+  --ntp                        \
+  --root-keys                  \
+  --repo-proxy                 \
+  --debug                      \
+  --keyfile $SSH_KEYFILE
```

Comment:
@erikPrime Good points.  These scripts are far from done, and we're re-building the automation around them.  I imagine we will continue to generalize them as necessary so they can be called by other scripts and various information passed in.

For now, we're running them by hand, but soon will be writing the scripts that configure all the various variables and then call these.

> I think we should CONSIDER using a git submodule to pull in the helper.rb instead of using copy pasta.

Our `helper.rb` will be unique - despite the name being the same in all the projects, they don't actually contain the same content.  I believe the naming is allude to the fact that it's the file you pass to the Beaker `--helper` argument.